src/Microsoft.PowerShell.Commands.Utility/resources/WebCmdletStrings.resx

```diff
@@ -249,6 +252,9 @@
   <data name="WebMethodInvocationVerboseMsg" xml:space="preserve">
     <value>{0} {1} with {2}-byte payload</value>
   </data>
+  <data name="WebMethodResumeFailedVerboseMsg" xml:space="preserve">
+    <value>The remote server indicated it could not resume downloading. This often occurs when the local file and remote file are different sizes. The local file will be overwritten.</value>
```

Comment:
I believe we could remove `This often occurs when the local file and remote file are different sizes.`

Reply:
Agree.  The extra detail may be confusing to novice users.

Reply:
Oddly enough.. I added this explanation because I thought it was confusing without it. I'll remove it. 😆

Reply:
fixed.